neqo-http3/src/connection.rs

```diff
@@ -323,6 +323,46 @@ impl<T: Http3Transaction> Http3Connection<T> {
         }
     }
 
+    pub fn is_critical_stream(&self, stream_id: u64) -> bool {
+        if let Some(id) = self.qpack_encoder.local_stream_id() {
```

Comment:
There is a clever way of doing this, in case you were interested...

```rust
self.qpack_encoder.local_stream_id().iter().chain(
  self.qpack_encoder.remote_stream_id().iter()
) // .chain ...
.any(|id| stream_id == id)
```

Reply:
Thanks. I like it. I like how much I have learned rust with your comments.